pyccel/parser/syntactic.py

```diff
@@ -385,191 +340,256 @@ def _visit_NoneType(self, stmt):
 
     def _visit_str(self, stmt):
 
-        return repr(stmt)
+        return stmt
 
-    def _visit_StringNode(self, stmt):
-        val =  stmt.value
-        if isinstance(stmt.parent,(RedBaron, DefNode)):
+    def _visit_Str(self, stmt):
+        val =  stmt.s
+        if isinstance(self._scope[-2], ast.Expr):
             return CommentBlock(val)
         return String(val)
```

Comment:
Quoting the python ast documentation (https://docs.python.org/dev/library/ast.html#node-classes)

> Changed in version 3.8: Class ast.Constant is now used for all constants.

> Deprecated since version 3.8: Old classes ast.Num, ast.Str, ast.Bytes, ast.NameConstant and ast.Ellipsis are still available, but they will be removed in future Python releases. In the meantime, instantiating them will return an instance of a different class.

So it looks like we should use `ast.Constant` and then check its value. What is our strategy for supporting multiple Python 3 versions in the future?

Reply:
Sorry, I have just realized that indeed we have a `_visit_Constant` function starting at line 419. Well done!